src/service/fixed-layer.js

```diff
@@ -295,6 +295,17 @@ export class FixedLayer {
   removeFixedElement_(element) {
     for (let i = 0; i < this.fixedElements_.length; i++) {
       if (this.fixedElements_[i].element == element) {
+        this.vsync_.run({
```

Comment:
Done - 

ZIndex is not needed its being handled in returnFromFixedLayer_ (which is where it is needed)